### ISSUE TITLE
VAV system with high flow rates and fan pressure hits max iterations with draw-thru fan

### ISSUE THREAD

**Issue by mjwitte** [NREL/EnergyPlus#6175]
Issue overview
--------------
User file is a data center with high internal gains, relatively warm supply temperature, high fan pressure and thus large fan power.  With the draw-thru fan, fixed supply temperature setpoint, and setpointmanager:mixedair to adjust the setpoint to the cooling coil, the system runs fine some of the time, but other time steps the cooling coil is way over or under the setpoint, and there are max iteration warnings.  The max iterations rotate thru the same four or five values repeatedly.  Setting the fan power to zero, or moving the fan to blow-thru eliminates the iteration warnings and the system controls exactly to setpoint.

### Workaround
Set the sensor node on the Controller:WaterCoil to be downstream past the fan, not the coil air outlet.  Ignore the warning that says you shouldn't do that.

### Details
Some additional details for this issue (if relevant):
 - Platform Win64
 - Version of EnergyPlus v8.7.0
 - Direct e-mail 6/13/2017

### Checklist
Add to this list or remove from it as applicable.  This is a simple templated set of guidelines.
 - [x] Defect file added EnergyPlusDevSupport\DefectFiles, Houston weather
 - [ ] Ticket added to Pivotal for defect (development team task)
 - [ ] Pull request created (the pull request will have additional tasks related to reviewing changes that fix this defect)

**Comment by mjwitte** [comment]
@EnergyArchmage @rraustad Looked at this a little today.  Tried moving `ManageSetPoints` inside the iteration loop in SimHVAC.  That helped a little to bring the airloop outlet temperature closer to setpoint, but things still bounced around and the iteration counts went way up.  Then I took the defect file and moved the coil controller sensor node from the coil outlet to the airloop outlet (simply changed one node name in the controller), and voila!  No max iterations, and the system purrs along like a kitten supplying exactly the 27C setpoint.  So, this warning is bad advice, at least for this application:
```
   ** Warning ** HVACControllers: GetControllerInput: Controller:WaterCoil="DATA CENTER CRAH CLG COIL CONTROLLER". 
   **   ~~~   **  ..Sensor node not found on water coil air outlet node.
   **   ~~~   **  ..The sensor node may have been placed on a node downstream of the coil or on an airloop outlet node.
```

**Comment by EnergyArchmage** [comment]
Yeah, the lag in the mixed air  SPM is probably a bigger deal for this case (high fan heat, draw though, no heating coil to fight with).  The coil  controller is more integrated into the calculations and can handle it better than mixed air SPM.
 Maybe the warning should be reworded to have a exception for draw-thru, high-pressure fans, cooling only AHU.
